### PR TITLE
Remove restricted presets on "amp-fx" attribute

### LINKED ISSUE #16956: amp-fx doesn't validate combined presets
E.g. 
`amp-fx="parallax fade-in"`

### PR DESCRIPTION
**Pull request by nainar**
Fixes #16956

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
I think there may be a better solution. This PR would allow any value to be present. Instead this should be changed to a regex that allows for combinations separated by whitespace. Note that we recently changed to this repeated field and it used to be:

```
value_regex_casei: "fade-in|"
                       "fade-in-scroll|"
                       "fly-in-bottom|"
                       "fly-in-left|"
                       "fly-in-right|"
                       "fly-in-top|"
                       "
```

**Comment by nainar** [comment]
@honeybadgerdontcare changed it to allow a space separated list of values. PTAL?

File: validator/validator-main.protoascii [hunk 1]
```diff
@@ -5344,13 +5344,7 @@ attr_lists: {
...
   attrs: {
     name: "amp-fx"
-    value_casei: "fade-in"
-    value_casei: "fade-in-scroll"
-    value_casei: "fly-in-bottom"
-    value_casei: "fly-in-left"
-    value_casei: "fly-in-right"
-    value_casei: "fly-in-top"
-    value_casei: "parallax"
+    value_regex_casei: "^(fade-in|fade-in-scroll|fly-in-bottom|fly-in-left|fly-in-right|fly-in-top|parallax)|(fade-in|fade-in-scroll|fly-in-bottom|fly-in-left|fly-in-right|fly-in-top|parallax)\\s+(fade-in|fade-in-scroll|fly-in-bottom|fly-in-left|fly-in-right|fly-in-top|parallax)"
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]:
```
    value_regex_casei: "(fade-in|"
                        "fade-in-scroll|"
                        "fly-in-bottom|"
                        "fly-in-left|"
                        "fly-in-right|"
                        "fly-in-top|"
                        "parallax)"
                       "(\\s|"
                        "fade-in|"
                        "fade-in-scroll|"
                        "fly-in-bottom|"
                        "fly-in-left|"
                        "fly-in-right|"
                        "fly-in-top|"
                        "parallax)*"
```

Comment (nainar) [reply validator/validator-main.protoascii]: Changed the regex, PTAL?

**Pull request merged**